modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/schema/CatalogSqlSchemaManager.java

```diff
@@ -70,7 +71,7 @@ public CatalogSqlSchemaManager(CatalogManager catalogManager, int cacheSize) {
 
     /** {@inheritDoc} */
     @Override
-    public SchemaPlus schema(String name, int version) {
+    public @Nullable SchemaPlus schema(String name, int version) {
```

Comment:
`String name` can be `null` not return value =)

Reply:
Ok.
Return value also can be null if someone use non-existed schema name.